### PR TITLE
[PSDK-443] Quickstart Basenames Registration

### PR DESCRIPTION
**Pull request by John-peterson-coinbase**
### What changed? Why?
- Add quick start demo for Basenames registration.

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by rohan-agarwal-coinbase — APPROVED** [review]
awesome!

**Comment by cb-heimdall** [comment]
Review Error for rohan-agarwal-coinbase @ 2024-09-12 21:28:42 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

File: quickstart-template/register-basename.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    console.error(
+      `Error loading funded wallet ${walletId} from seed file ${seedFilePath}: `,
+      error,
+    );
+  }
+}
+
+async () => {
+  try {
+    const { BASE_NAME, WALLET_ID, SEED_FILE_PATH } = process.env;
```
Comment (jazz-cb) [inline quickstart-template/register-basename.js:147]: add a comment that it should be set in ?

**Review by rohan-agarwal-coinbase — APPROVED** [review]

**Review by jazz-cb — APPROVED** [review]

File: quickstart-template/register-basename.js (hunk 1 above)
Comment (John-peterson-coinbase) [reply quickstart-template/register-basename.js:147]: The Solution guide will have a section explaining how to set these.

**Pull request merged**